.github/workflows/workflow_terraform_environment_cleanup.yml

```diff
@@ -53,5 +53,7 @@ jobs:
 
       - name: Destroy PR Terraform Workspaces
         working-directory: ./terraform/environment
+        env:
```

Comment:
Was it just the case that the script failed because it couldn't communicate with pagerduty?

Reply:
Yeah, the terraform won't run without a pagerduty_token Terraform variable being passed to it